.github/ISSUE_TEMPLATE/5.priority-support.yml

```diff
@@ -7,20 +7,22 @@ body:
     attributes:
       value: |
         Please provide a searchable summary of the issue in the title above ⬆️.
-  - type: checkboxes
+
+  - type: input
     attributes:
-      label: Duplicates
-      description: Please [search the history](https://github.com/mui/mui-x/issues) to see if an issue already exists for the same problem.
-      options:
-        - label: I have searched the existing issues
-          required: true
+      label: Search keywords
+      description: Your issue may already have been reported. Please list the keywords you've used to search the [existing issues](https://github.com/mui/mui-x/issues). This will allow us to improve the experience for other users.
```

Comment:
```suggestion
      description: Your issue may have already been reported. Please list the keywords you've used to search the [existing issues](https://github.com/mui/mui-x/issues). This will allow us to improve the experience for other users.
```